packages/runtime/container-runtime/src/summary/summarizerNode/summarizerNode.ts

```diff
@@ -433,17 +433,6 @@ export class SummarizerNode implements IRootSummarizerNode {
 							summaryRefSeq,
 						};
 					}
-
-					const props = {
-						summaryRefSeq,
-						pendingSize: this.pendingSummaries.size ?? undefined,
-					};
-					this.logger.sendTelemetryEvent({
-						eventName: "PendingSummaryNotFound",
-						proposalHandle,
-						referenceSequenceNumber: this.referenceSequenceNumber,
-						details: JSON.stringify(props),
```

Comment:
Unrelated side note - anywhere we do JSON.stringify in a `sendTelemetryEvent` call we can just log the object directly now and the logger will stringify it.

Reply:
Can you show me an example? I would like to do that for a few things.

Reply:
In this case literally just delete `JSON.stringify` (and the parens) so it's `details: props`.  The result will be the same.

Reply:
I tried changing the properties for the `refreshLatestSummary` function to be in details but it did not work. I will reach out to you regarding that.

Reply:
As discussed, the problem is the PerfEvent APIs weren't updated to the new Ext types.  I'll try to slip in a fix there soon.